packages/process/src/node/dev-null-stream.ts

```diff
@@ -22,6 +22,19 @@ import stream = require('stream');
  * Writing goes to a black hole, reading returns `EOF`.
  */
 export class DevNullStream extends stream.Duplex {
+
+    constructor(options: {
+        /**
+         * Makes this stream call `destroy` on itself, emitting the `close` event.
+         */
+        autoDestroy?: boolean,
```

Comment:
When is it `false`? I cannot find a single client.

Reply:
I just don't feel confident making this the default behavior. I prefer this action to be explicitly defined by clients, it is mostly syntactic here.

It allows to write:

```ts
this.outputStream = this.process.stdout || new DevNullStream({ autoDestroy: true });
```

When it would be the following if it did not have the option:

```ts
if (this.process.stdout) {
   this.outputStream = this.process.stdout;
} else {
    this.outputStream = new DevNullStream();
    this.outputStream.destroy();
}
```

Reply:
ok